loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/JsonLoadFlowParameters.java

```diff
@@ -43,20 +43,15 @@
     public interface ExtensionSerializer<E extends Extension<LoadFlowParameters>> extends ExtensionJsonSerializer<LoadFlowParameters, E> {
     }
 
-    /**
-     * Lazily initialized list of extension serializers.
-     */
-    private static final Supplier<ExtensionProviders<ExtensionSerializer>> SUPPLIER =
-            Suppliers.memoize(() -> ExtensionProviders.createProvider(ExtensionSerializer.class, "loadflow-parameters"));
-
-    /**
-     * Gets the known extension serializers.
-     */
-    public static ExtensionProviders<ExtensionSerializer> getExtensionSerializers() {
-        return SUPPLIER.get();
+    private JsonLoadFlowParameters() {
     }
 
-    private JsonLoadFlowParameters() {
+    public static Map<String, ExtensionJsonSerializer> getExtensionSerializers() {
+        List<LoadFlowProvider> providers = new ServiceLoaderCache<>(LoadFlowProvider.class).getServices();
+        return providers.stream()
+            .filter(loadFlowProvider -> loadFlowProvider.getParametersExtensionSerializer().isPresent())
```

Comment:
I think you can use a `flatMap(loadFlowProvider -> loadFlowProvider.getParametersExtensionSerializer().stream())` for shorter syntax and avoid to use a raw `get()` on the optional later